miio/integrations/petfeeder/device.py

```diff
@@ -0,0 +1,313 @@
+        """Dispense food.
+
+        Manually dispense food.
+
+        :param int units: Number of units to dispense (1 unit ~= 5g)
+        :return: MiIO response
+        :rtype: bool
+        """
+        return self.send("outfood", [units]) == OK
+
+    @command(default_output=format_output("Resetting desiccant time left"))
+    def reset_desiccant_left(self) -> bool:
+        """Reset desiccant time.
+
+        Reset the desiccant life counter (30 days)
+
+        :return: MiIO response
+        :rtype: bool
+        """
+        date = (datetime.now() + timedelta(days=30)).strftime("%Y-%m-%d")
+        return self.send("resetdryer", [date]) == OK
+
+    @command(
+        click.argument("state", type=int),
+        default_output=format_output(
+            lambda state: "Turning on WiFi led" if state else "Turning off WiFi led"
+        ),
+    )
+    def set_wifi_led(self, state: bool) -> bool:
+        """Set wifi led.
+
+        Enable / Disable the wifi status led.
+
+        :param bool state: On / Off
+        :return: MiIO response
+        :rtype: bool
+        """
+        return self.send("wifiledon", [int(state)]) == OK
+
+    @command(
+        click.argument("state", type=int),
+        default_output=format_output(
+            lambda state: "Locking dispense button"
+            if state
+            else "Unlocking dispense button"
+        ),
+    )
+    def set_button_lock(self, state: bool) -> bool:
+        """Set button lock.
+
+        Lock / Unlock the manual dispense button.
+
+        :param bool state: Unlocked / Locked
+        :return: MiIO response
+        :rtype: bool
+        """
+        return self.send("keylock", [int(state) ^ 1]) == OK
+
+    @command(
+        click.argument("state", type=int),
+        default_output=format_output(
+            lambda state: "Enabling automatic feeding schedule"
+            if state
+            else "Disabling automatic feeding schedule"
+        ),
+    )
+    def set_feed_state(self, state: bool) -> bool:
+        """Set feed state.
+
+        Enable / Disable the automatic feeding schedule.
+
+        :param bool state: Off / On
+        :return: MiIO response
+        :rtype: bool
+        """
+        return self.send("stopfeed", [int(state)]) == OK
+
+    @command(
+        default_output=format_output(
+            "",
+            lambda result: "".join(
+                (
+                    f"Slot: {plan.slot:02d} "
+                    f"Time: {plan.datetime.astimezone(PetFeeder._local_tz).strftime('%H:%M')} "
+                    f"Units: {plan.units}\n"
+                )
+                for plan in result
+                if plan.enabled
+            ),
+        ),
+    )
+    def get_feed_plans(self) -> List[FeedPlan]:
+        """Get feed plans.
+
+        Return all feeding plans.
+
+        return: List of FeedPlan objects
+        rtype: list
+        raises PetFeederException: if the number of elements returned is wrong
```

Comment:
```suggestion
        :return: List of FeedPlan objects
        :rtype: list
        :raises PetFeederException: if the number of elements returned is wrong
```

Reply:
Actually, see my previous comment. If there's no very specific need to document any of these, please remove it and just annotate the method accordingly. I think only exceptions cannot be expressed nicely using bare type hints..